### PR TITLE
Transcribe SmartRockets to Processing.

### PR DESCRIPTION
**Pull request by FriskyDev**
Ok, I took a few liberties with this one. If you would like, I can transcribe this closer to the actual video.

### CODE REVIEW

**Comment by gruselhaus** [comment]
Thank you for your help!

File: CodingChallenges/CC_029_SmartRockets/Processing/CC_029_SmartRockets/Dna.pde [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// http://patreon.com/codingtrain
+// Code for: https://youtu.be/bGz7mv2vD6g
+// Processing transcription: Chuck England
+
+import java.util.*;
+
+int lifespan = 400;
+
+class DNA {
+  List<PVector> genes;
```
Comment (shiffman) [inline CodingChallenges/CC_029_SmartRockets/Processing/CC_029_SmartRockets/Dna.pde]: Would a plain array be simpler to follow here given the lifespan is fixed?

Comment (FriskyDev) [reply CodingChallenges/CC_029_SmartRockets/Processing/CC_029_SmartRockets/Dna.pde]: I’ll take a look again. But I was thinking about the previous comments when I did it, and I think the genes were dynamic.

Comment (FriskyDev) [reply CodingChallenges/CC_029_SmartRockets/Processing/CC_029_SmartRockets/Dna.pde]: Ok, you were right, doh! I guess I saw the .add() I used and thought it was dynamic. Updated.

Comment (FriskyDev) [reply CodingChallenges/CC_029_SmartRockets/Processing/CC_029_SmartRockets/Dna.pde]: image

**Pull request merged**